test/metabase/api/collection_test.clj

```diff
@@ -1488,53 +1521,53 @@
     (testing "I shouldn't be allowed to move the Collection without proper perms."
       (testing "If I want to move A into B, I should need permissions for both A and B"
         (mt/with-non-admin-groups-no-root-collection-perms
-          (mt/with-temp* [Collection [collection-a]
-                          Collection [collection-b]]
-            (perms/grant-collection-readwrite-permissions! (perms-group/all-users) collection-a)
-            (is (= "You don't have permissions to do that."
-                   (mt/user-http-request :rasta :put 403 (str "collection/" (u/the-id collection-a))
-                                         {:parent_id (u/the-id collection-b)}))))))
+         (mt/with-temp* [Collection [collection-a]
+                         Collection [collection-b]]
+           (perms/grant-collection-readwrite-permissions! (perms-group/all-users) collection-a)
+           (is (= "You don't have permissions to do that."
+                  (mt/user-http-request :rasta :put 403 (str "collection/" (u/the-id collection-a))
+                                        {:parent_id (u/the-id collection-b)}))))))
 
       (testing "Perms checking should be recursive as well..."
         (testing "Create A, B, and C; B is a child of A."
           (testing "Grant perms for A and B. Moving A into C should fail because we need perms for C"
             ;; (collections with readwrite perms marked below with a `*`)
             ;; A* -> B* ==> C -> A -> B
             (mt/with-non-admin-groups-no-root-collection-perms
```

Comment:
The previous indentation is correct  https://guide.clojure.style/#body-indentation

Reply:
Fixed